Components/Modal/modal.css

```diff
@@ -0,0 +1,66 @@
+.modal-box {
+  border: 1px solid var(--gray);
+  margin: auto;
+  /* width:80%; */
+  padding: 0.3rem 1rem;
+  border-radius: 6px;
+  margin-bottom: 1.5rem;
+}
+
+.modal-container {
+  margin: 2rem 5rem;
+  background: var(--blue-border);
+  width: 25rem;
+  padding: 12px;
+}
+.modal-header-btn {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 40px;
+  padding: 10px;
+}
+
+.modal-header {
+  font-size: 1.5rem;
+}
+.modal-close {
+  cursor: pointer;
+  background: var(--blue-border);
+  outline: none;
+  font-size: 1.2rem;
+  padding: 2px 4px;
+}
+.modal-close:hover {
+  border: 2px solid var(--black-bg);
+}
+.modal-product {
+  margin: 2px;
+}
+.modal-footer {
+  display: flex;
+  height: 60px;
+  align-items: center;
+  justify-content: flex-end;
+}
+.modal-footer button:first-child {
+  margin-right: 5px;
+}
+.btn-cancel:hover {
+  opacity: 0.7;
+}
+#example-modal-CTA {
+  margin: 1rem;
+}
+#example-modal.show {
+  opacity: 1;
+  pointer-events: auto;
+  margin: 2rem 5rem;
+  background: var(--blue-border);
+  width: 25rem;
+  padding: 12px;
```

Comment:
avoid using id and combinations of selectors with high specificty it might cause trouble in the longer run.